index.html

```diff
@@ -3110,6 +3110,12 @@ <h1 class="large-h1">Subscribe to stay informed</h1>
         });
 
         });
-      </script>
+        </script>
```

Comment:
No need to separate out this script section from the one below it. Please remove this closing script tag.